test/deprecated/string-isEmptyString.chpl

```diff
@@ -0,0 +1 @@
+var b = 'deprecate me please'.isEmptyString();
```

Comment:
Kinda silly, but maybe test on an empty and non-empty string just to make sure the deprecated thing still works until it's removed?

Reply:
sure